hasura/graphql-engine
# add pagination on rawsql results page (close #5002)

### Description
this PR will add pagination feature on rawSQL results page which makes console more performant, by doing this large data will not be displayed in a single big table, instead the pagination take care of the data rendering.

### Changelog


### Affected components



### Related Issues
close #5002  

### Solution and Design
added pagination using react table 

### Steps to test and verify
Run a SQL query that has huge number of results and verify the pagination feature.
